File: frontend/src/lib/types/person.ts
export interface Person {
  id: number;
  name: string;
  email: string;
  availableDays: number;
}

export interface PersonFormData {
  name: string;
  email: string;
  availableDays: number;
}

export type CreatePersonDto = PersonFormData;
export type UpdatePersonDto = PersonFormData;

File: backend/Descansario.Api/Models/Person.cs
namespace Descansario.Api.Models;

public class Person
{
    public int Id { get; set; }

    public required string Name { get; set; }

    public required string Email { get; set; }

    public int AvailableDays { get; set; } = 20; // Días de vacaciones disponibles al año

    // Navigation property
    public ICollection<Vacation> Vacations { get; set; } = new List<Vacation>();
}
